File: VB/AxisTitleHitTestSample/SampleForm.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Windows.Forms

Namespace AxisTitleHitTestSample
    Partial Public Class SampleForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Overloads Sub OnMouseMove(ByVal sender As Object, ByVal e As MouseEventArgs) Handles chart.MouseMove
            Dim hitInfo As ChartHitInfo = chart.CalcHitInfo(e.X, e.Y)
            If hitInfo.AxisTitle IsNot Nothing Then
                tooltipController.ShowHint(String.Format("The pointer is over the {0} object.", hitInfo.AxisTitle.GetType().ToString()))
            Else
                tooltipController.HideHint()
            End If
        End Sub
    End Class
End Namespace

File: CS/AxisTitleHitTestSample/SampleForm.cs
using DevExpress.XtraCharts;
using System;
using System.Windows.Forms;

namespace AxisTitleHitTestSample {
    public partial class SampleForm : Form {
        public SampleForm() {
            InitializeComponent();
        }

        private void OnMouseMove(object sender, MouseEventArgs e) {
            ChartHitInfo hitInfo = chart.CalcHitInfo(e.X, e.Y);
            if (hitInfo.AxisTitle != null)
                tooltipController.ShowHint(
                    String.Format(
                        "The pointer is over the {0} object.", 
                        hitInfo.AxisTitle.GetType().ToString()
                    )
                );
            else
                tooltipController.HideHint();
        }
    }
}
